File: cms/src/main/java/com/imps/cms/model/dto/UserDto.java
package com.imps.cms.model.dto;


import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;
import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class UserDto implements Serializable {
    private Long id;
    private String fullName;
    private String salt;
    private String email;
    private String password;
    private Long sectionId;
    private List<Long> proposalIds;
    private boolean activated;
}

File: cms/src/main/java/com/imps/cms/angular/src/app/authentication/model/UserDto.ts
export class UserDto {
  id = -1;
  fullName: string;
  salt = 'salt';
  email: string;
  password: string;
  sectionId = -1;

  constructor(email, fullName, password) {
    this.email = email;
    this.fullName = fullName;
    this.password = password;
  }
}
